pkg/controller/keepalivedgroup/keepalivedgroup_controller.go

```diff
@@ -293,16 +293,18 @@ func findNextAvailableID(ids []int) (int, error) {
 	if len(ids) == 0 {
 		return 1, nil
 	}
-	if len(ids) > 255 {
-		return 0, errors.New("cannot allocate more than 255 ids in one keepalived group")
-	}
 	sort.Ints(ids)
```

Comment:
is this needed? at this point I'd say not.

Reply:
also the sort at line 276 is probably not needed.

Reply:
Updated